doc/changelog/02-specification-language/11132-master+fix-implicit-let-fixpoint-bug3282.rst

```diff
@@ -0,0 +1 @@
+- Fixed bugs sometimes preventing to define valid (co)fixpoints with implicit arguments in the presence of local definitions, see #3282 (`#11132 <https://github.com/coq/coq/pull/11132>`_, by Hugo Herbelin).
```

Comment:
The suggestion is mostly to add a link:

```suggestion
- Fixed bugs sometimes preventing to define valid (co)fixpoints with implicit arguments
  in the presence of local definitions, see `#3282 <https://github.com/coq/coq/issues/3282>`_
  (`#11132 <https://github.com/coq/coq/pull/11132>`_, by Hugo Herbelin).
```

I agree with Emilio. Including an example in the changelog would be great. Feel free to use the `.. example::` and the `.. coqtop::` directives.